### ISSUE TITLE
Group order bug

### ISSUE THREAD

**Issue by semy** [namespace-ee/react-calendar-timeline#76]
Hi,

thank you for your awesome timeline plugin, it works pretty nice :-)

I have found a bug in your group orders.

If I pass the groups sorted by id 1,2,3,4 the timeline works.

But If I sort my groups by name, the items are no longer classified into their respective groups.

This...

```
groups: [
   { id: 1, ...}
   { id: 2, ...}
   { id: 3, ...}
   { id: 4, ...}
]
```

... works, and this...

```
groups: [
   { id: 3, ...}
   { id: 1, ...}
   { id: 4, ...}
   { id: 2, ...}
]
```

... don't work!

It would be great if you could fix the problem.

Thank you and many greets
Semy

**Comment by schwermer** [comment]
I also experienced this same issue.  Item top dimensions are computed incorrectly if the groups are not in ascending order by ID.  I've submitted a PR, hopefully it gets merged soon

**Comment by signalwerk** [comment]
@schwermer I made new version for the component. In version 0.11.2 I only applied your patch for this bug plus fixed some styling issues of your code to match the eslint rules. I also wrote a quick test in `/src/lib/__tests__/` you can run it with `npm run jest`

here you find the pre-release: [group-ordering branch](https://github.com/signalwerk/react-calendar-timeline/tree/hotfix/group-ordering) do you think I can merge it to master?

**Comment by signalwerk** [comment]
thanks to @schwermer this is corrected in Version 0.11.2. now on master.
